pkg/domain/podman/router.go

```diff
@@ -106,13 +111,17 @@ func (r *RouterEntityManager) QueryAllRouters() ([]qdr.Router, error) {
 		var records []qdr.Record
 		err = json.Unmarshal([]byte(rJson), &records)
 		if err != nil {
+			// routerToQuery no longer exists
+			if routerToQuery != "" {
```

Comment:
I don't follow the logic here still. When would the router id be ""? And why does the error to be returned include the router id, if it will only be returned if the id is ""?

Reply:
The logic is not valid. I have updated it to ignore both SendException and Timeout messages.